Kickstarter-iOS/Views/Controllers/RewardsCollectionViewController.swift

```diff
@@ -327,22 +244,6 @@ extension RewardsCollectionViewController {
   }
 }
 
-// MARK: - UIScrollViewDelegate
-
-extension RewardsCollectionViewController {
-  override func scrollViewDidScroll(_ scrollView: UIScrollView) {
-    self.collectionView.visibleCells.compactMap { $0 as? RewardCell }.forEach {
-      $0.cancelDepress()
```

Comment:
Is `cancelDepress` still being used?

Reply:
Oh good catch. Hmm @justinswart can you describe the effect of this? I'm not seeing any issues when removing it, the "depress" action seems to cancel when you lift up your finger.

Reply:
This was for when you begin dragging the carousel, Danny wanted the card to lift again as you pan. I was mostly indifferent about it.